openedx/core/djangoapps/credentials_service/api/v1/views.py

```diff
@@ -0,0 +1,116 @@
+"""
+Credentials service API views (v1).
+"""
+import logging
+from openedx.core.djangoapps.credentials_service import serializers
+from openedx.core.djangoapps.credentials_service import filters
+from openedx.core.djangoapps.credentials_service.models import UserCredential, ProgramCertificate, CourseCertificate, \
+    UserCredentialAttribute
+from openedx.core.lib.api import parsers
+from rest_framework import mixins, viewsets, parsers as drf_parsers
+
+from rest_framework.response import Response
+from rest_framework import status
+
+log = logging.getLogger(__name__)
+
+
+class UserCredentialViewSet(mixins.CreateModelMixin, mixins.ListModelMixin,
+    mixins.RetrieveModelMixin, mixins.UpdateModelMixin, viewsets.GenericViewSet):
+
+    queryset = UserCredential.objects.all()
+    lookup_field = 'username'
+    serializer_class = serializers.UserCredentialSerializer
+    parser_classes = (parsers.MergePatchParser,  drf_parsers.JSONParser)
+
+    def partial_update(self, request, *args, **kwargs):
+        """
+        PATCH /api/credentials/v1/users/{username}/
+        Only to update the certificate status.
+        """
+        UserCredential.objects.filter(pk=request.data.get('id')).update(
+            status=request.data.get('status')
+        )
+
+        return Response([], status=status.HTTP_200_OK)
```

Comment:
is this a placeholder?  it might be more convenient for clients if you return the content of the updated object.